client-tools/CMakeLists.txt

```diff
@@ -380,6 +380,7 @@ add_executable(${BIN_ARANGOSH}
   Shell/V8ClientConnection.cpp
   Shell/V8ShellFeature.cpp
   Shell/arangosh.cpp
+  Shell/RequestFuzzer.cpp
```

Comment:
Should only be added if cmake variable `USE_FAILURE_TESTS` is set.

Reply:
Addressed